### PR TITLE
Re-design of playbooks

### PR DESCRIPTION
**Pull request by vidvidvid**
https://www.figma.com/file/IzgwXeC5eDVo7ayAmYihwS/Combine-Paths?node-id=268%3A2477&mode=dev

### CODE REVIEW

**Comment by peth-yursick** [comment]
I can't see if the design was applied to all 3, so thought I should point that out just in case - since "paths", "playbooks" & "great houses" are all "academy" now - this design should apply to all 3.


Also reminds me of the links:

While doing that, might also make sense to move them together.
Right now we got:
Playbooks - metagame.wtf/learn/playbooks/meme-driven-organizations
Great houses - metagame.wtf/learn/great-houses/house-of-ethereum
Paths - metagame.wtf/play/paths/engaged-octos-path

Should all be:
Playbooks - metagame.wtf/academy/meme-driven-organizations
Great houses - metagame.wtf/academy/house-of-ethereum
Paths - metagame.wtf/academy/engaged-octos-path

In short, all of these:
/learn/playbooks/
/learn/great-houses/
/play/paths/
Should be replaced with just: /academy/

Depending on how much additional work this requires, we can make it into a new issue, up to you.

**Comment by peth-yursick** [comment]
Alright, to recap:
[ ] mobile, guess you knew that but just in case
[ ] submit button button not the same color as connect & other buttons
[ ] top slot in the sidebar has rounded corners, should be straight like others
[ ] need to apply this design to paths & great houses as well
[ ] (optional) move playbooks, great houses & paths from /learn/playbooks/, /learn/great-houses/ & /play/paths/ to just: /academy/
[ ] (this would also fix the paths being broken)
[ ] currently, the top item on the sidebar has round corners & all others have straight. The topmost should have round top corners & bottomost should have round bottom corners, others are straight

File: packages/web/components/QuestChain/UploadProof.tsx [hunk 1]
```diff
@@ -185,87 +205,33 @@ export const UploadProof: React.FC<{
...
       >
         <StatusedSubmitButton
+          px={[8, 12]}
+          label="Submit Proof"
           onClick={() => {
-            setIsSubmittingProof(true);
-            onOpen();
+            if (!chainId || chainId !== questChain.chainId || !provider) {
+              addToast({
+                description: `Wrong Chain, please switch to ${
```
Comment (Seroxdesign) [inline packages/web/components/QuestChain/UploadProof.tsx:213]: You're handling 3 separate cases and the description only covers 1, maybe change the description to something like: "Please ensure your wallet is connected to the x chain."

**Review by Seroxdesign — CHANGES_REQUESTED** [review]

**Review by Seroxdesign — APPROVED** [review]
